### DISCUSSION TITLE
C# HttpClient

### DISCUSSION THREAD

**Discussion by ajdust** [Kong/httpsnippet#275] [New Client Request]
Hi,

It looks like only RestSharp is here for C#. [HttpClient](https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient?view=netcore-2.2) is in the .NET standard library and is the standard way to make HTTP requests (without a third party library like RestSharp).

If there is no ongoing work to add HttpClient, I can PR to add a snippet for HttpClient.

Thanks.

**Comment by darrenjennings** [comment]
Would gladly welcome this, thanks @johnsabr.

**Comment by h4ck1th4rd** [comment]
Any news for adding C# HttpClient?, Thank you.
